viewing_party/main.py

```diff
@@ -0,0 +1,120 @@
+def create_movie(title, genre, rating):
+    new_movie = {}
+    if title and genre and rating:
+        new_movie["title"] = title 
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie
```

Comment:
Nice! Another way to write this would be:

new_movie = {
        "title": title,
        "genre", genre,
       "rating", rating
}